tests/python/gaia-ui-tests/gaiatest/apps/ftu/app.py

```diff
@@ -334,9 +334,20 @@ def a11y_click_next_to_welcome_browser_section(self):
             Wait(self.marionette).until(expected.element_present(
                 *self._section_welcome_browser_locator))))
 
+    @property
+    def is_statistics_enabled(self):
+        return self.marionette.find_element(
+            *self._statistic_checkbox_locator).is_selected()
+
     def tap_statistics_checkbox(self):
         self.marionette.find_element(*self._enable_statistic_checkbox_locator).tap()
 
+    def toggle_statistics(self):
```

Comment:
This implementation is way better than `tap_statistics_checkbox`.

One thing though. This is debatable, but I would prefer to have functions in the page class which don't depend on the state of the UI. 
The main reason is it makes the test more deterministic: If I call twice the same method, the same thing will happen on the UI. Also, it makes the expectation of the test writer clearer, like: "If I use `ftu.enable_send_statistics()`, I expect a active checkbox after this function call."

What do you think?